recipes/reprit/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "reprit" %}
+{% set version = "0.3.0" %}
```

Comment:
as far as I understood required LICENSE file is included in package only starting from [version `0.3.1`](https://pypi.org/project/reprit/0.3.1/)

Reply:
I have updated reprit to `0.3.1` and wagyu to `0.2.2`